boost-maven-plugin/src/main/java/boost/project/LibertyBoostMojo.java

```diff
@@ -172,6 +225,32 @@ private ExecutionEnvironment getExecutionEnvironment() {
         return executionEnvironment(mavenProject, mavenSession, pluginManager);
     }
 
+    /**
+     * 
+     * @param boosterConfigurators
+     * @return
+     */
+    private List<String> getBoosterFeatureNames(List<BoosterPackConfigurator> boosterConfigurators) {
+			List<String> featureStrings = new ArrayList<String>();
+			for (BoosterPackConfigurator bpconfig : boosterConfigurators){
+				featureStrings.add(bpconfig.getFeatureString());
+			}
+			
+			return featureStrings;
+		}
+
+	private List<String> findJ2EEAppFeatureDependencies(MavenProject project) {
```

Comment:
Might be good to break some of these J2ee specific methods into a J2EEProjectUtils class. Although maybe that's for another item if/when we refactor into a more generic configurator between spring and j2ee

Reply:
same as my previous response above